src/ios-deploy/ios-deploy.m

```diff
@@ -268,13 +268,16 @@ CFStringRef copy_find_path(CFStringRef rootPath, CFStringRef namePattern) {
         CFRelease(path);
         return NULL;
     }
```

Comment:
It seems like using maxdepth makes this earlier speedup unnecessary. We should probably remove it.